dot/rpc/modules/chain.go

```diff
@@ -36,8 +36,11 @@ type ChainBlockNumberRequest struct {
 	Block interface{}
 }
 
-// ChainIntRequest represents an integer
-type ChainIntRequest uint64
+// ChainFinalizedHeadRequest ...
+type ChainFinalizedHeadRequest struct {
+	Round uint64
+	SetID uint64
+}
```

Comment:
will this require a change to the way the json-rpc call in formatted in `tests/utils/chain.go` `GetFinalizedHeadByRound`?

Reply:
Yes. It will be correctly parsed in the custom Unmarshall function that was added.